test/app-luatest/console_debugger_session_test.lua

```diff
@@ -267,21 +256,21 @@ local sequences = {
     },
 }
 
-local function run_debug_session(cmdline, sequence, tmpfile, header)
+local function run_debug_session(sequence, tmpfile, flags)
     local short_src = fio.basename(tmpfile)
     --[[
         repeat multiple times to check all command aliases
     ]]
     for i = 1, MAX_ALIASES_COUNT do
-        local fh = popen.new(cmdline, {
-            stdout = popen.opts.PIPE,
-            stderr = popen.opts.PIPE,
-            stdin = popen.opts.PIPE,
-        })
-        t.assert_is_not(fh, nil)
-        -- for -d option we expect debugger banner in the stderr,
-        -- but for self invoke debugger there is no extra header.
-        local first = header ~= nil
```

Comment:
I removed the check because it is hard to implement using `interactive_tarantool`. `it` has stderr_logger being run in a different fiber and I couldn't get an after-test check for empty stderr working in reasonable time. As we don't expect anything from stderr, and get all the right stdout output, it seems like the test is comprehensive enough.

Reply:
Ok, I've had another go at this, and now it seems to be working fine. `read_until_prompt()` in `interactive_tarantool` now expects empty stderr, while `new_debugger()` expects debugger header, similar to the old test.

Reply:
Thanks!